projects/igniteui-angular/src/lib/icon/types.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
+import { Theme } from 'igniteui-webcomponents/theming/types.js';
```

Comment:
Um, need to check if this gets compiled away entirely in the declarations, otherwise it'll make `igniteui-webcomponents` a direct dependency of the library which it isn't.
Also also, there's an identical `IgxTheme` in the theme service expored already which can be used instead, no?